pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -342,14 +355,6 @@ public long getLookupTimeoutMs() {
         }
     }
 
-    public ClientConfigurationData clone() {
-        try {
-            return (ClientConfigurationData) super.clone();
-        } catch (CloneNotSupportedException e) {
-            throw new RuntimeException("Failed to clone ClientConfigurationData");
-        }
-    }
-
```

Comment:
Is this change necessary?

Reply:
I move this method to the end of file. Make all the `get methods` together for better reading.

Reply:
I think we should keep this method where it is. We don't have a style guide covering this situation (that I know of), so I think we should keep things as they are until something more formal is decided on the mailing list.